Fig.Tests/JsonProviderTests.cs

```diff
@@ -55,5 +55,29 @@ public void CanResolveEnvironmentQualifiedSection()
             Assert.AreEqual("2", _settingsDictionary["EnvQualified.a:TEST"]);
             Assert.AreEqual("2", _settingsDictionary["EnvQualified.b:TEST"]);
         }
+
+        [Test]
+        public void CanGetArrayFromJsonArrayProperty()
+        {
+            var settings = new SettingsBuilder()
+                .UseSettingsDictionary(_settingsDictionary)
+                .Build<AppSettings>(prefix: "");
+
+            var servers = settings.Get<string[]>("Servers");
+            Assert.AreEqual(2, servers.Length);
+            Assert.AreEqual(new string[] { "10.0.0.1", "10.0.0.2" }, servers);
```

Comment:
The tests are passing so apparently this works..  I would have guessed that comparison is by reference. Some NUnit magic I suppose...

Reply:
I was surprised by this aswel at first, but it seems to compare the elements instead of the enumerable object. https://github.com/nunit/nunit/issues/2097